vmm/src/cpu.rs

```diff
@@ -342,12 +342,43 @@ pub struct CpuManager {
     vcpus_pause_signalled: Arc<AtomicBool>,
     reset_evt: EventFd,
     threads: Vec<thread::JoinHandle<()>>,
+    selected_cpu: u8,
 }
 
+const CPU_ENABLE_FLAG: usize = 1;
```

Comment:
ISTR that `CPEN` is bit 0, but I may be wrong...

Reply:
Oops, that's because I changed my mind and went from value to flag.